packages/react/src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -701,13 +701,15 @@ const DateTimePicker = ({
         setIsCustomRange(true);
         setCustomRangeKind(PICKER_KINDS.ABSOLUTE);
         if (!absolute.hasOwnProperty('start')) {
-          absolute.start = dayjs(absolute.startDate).valueOf();
+          absolute.start = dayjs(`${absolute.startDate} ${absolute.startTime}`).valueOf();
         }
         if (!absolute.hasOwnProperty('end')) {
-          absolute.end = dayjs(absolute.endDate).valueOf();
+          absolute.end = dayjs(`${absolute.endDate} ${absolute.endTime}`).valueOf();
         }
         absolute.startDate = dayjs(absolute.start).format('MM/DD/YYYY');
+        absolute.startTime = dayjs(absolute.start).format('HH:mm');
```

Comment:
Seem wrong to me that we are hard coding the date/time formats.  We should be looking at a way to externize this.  I don't think it is simple enough to expose a "pattern" since, what is the format of the pattern?  we can't have different components using different kinds of patterns... some using moment, some using dayjs, etc.

Reply:
I think PAL needs to have a standardize set of of API around globalization and that object should be made available to components that need to parse/format data, get access to masks, etc.

In graphite when we use "formats" we use named formats like "short_date", "long_date", "date_fully_month", etc, we translate those into the appropriate masks based on user locale.

Reply:
Thanks, Sean. We've talked about this internally and have started creating a document defining data formatting, i18n, data loading, etc--all the main items we've talked about over the last few months. We're moving that direction. This particular component was written 2 years ago, and this PR is to add time validation to the component as it already exists. The refactors based on new technical design will come later.